src/org/parosproxy/paros/control/MenuFileControl.java

```diff
@@ -164,6 +169,32 @@ public void newSession(boolean isPromptNewSession) throws ClassNotFoundException
 				break;
 		}
 	}
+
+	private boolean informStopActiveActions() {
+		String activeActions = wrapEntriesInLiTags(control.getExtensionLoader().getActiveActions());
+		if (!activeActions.isEmpty()) {
+			String message = MessageFormat
+					.format(Constant.messages.getString("menu.file.session.activeactions"), activeActions);
+			if (view.showConfirmDialog(message) != JOptionPane.OK_OPTION) {
+				return false;
+			}
+		}
+		return true;
+	}
+
+	private static String wrapEntriesInLiTags(List<String> entries) {
+		if (entries.isEmpty()) {
+			return "";

```

Comment:
Ok I was thinking more of some sort of weird race or time of check/time of use issue where we check something is running but then by the time it's added to the list it's done....

All good.